packages/eslint/common/no-external-imports/index.js

```diff
@@ -0,0 +1,84 @@
+// require('./typings.d.ts');
+const path = require('path');
+const fs = require('fs');
+
+const DEPENDENCIES_VARIANTS = ['dependencies', 'devDependencies', 'peerDependencies', 'optionalDependencies'];
+
+module.exports = (providedPath, moduleDependencies) => {
```

Comment:
Please change `providedPath` on more distinct name. Which exactly this path?

Reply:
fixed